internal/klocust/apply.go

```diff
@@ -0,0 +1,119 @@
+package klocust
+
+import (
+	"errors"
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"strings"
+
+	"github.com/DevopsArtFactory/klocust/internal/kube"
+	"github.com/DevopsArtFactory/klocust/internal/schemas"
+	"github.com/DevopsArtFactory/klocust/internal/util"
+	"k8s.io/klog/v2"
+)
+
+func checkInitFileNotFound(filenames []string, locustName string) error {
+	for _, filename := range filenames {
+		if isExist := util.IsFileExists(filename); !isExist {
+			return errors.New(fmt.Sprintf(
+				"`%s` file not found. \nYou need to init first before apply.\n\n$ klocust init %s\n",
+				filename, locustName))
+		}
```

Comment:
You can return err with formatting by using fmt.errorf 

--
`fmt.Errorf("`%s` file not found. \nYou need to init first before apply.\n\n$ klocust init %s\n", filename, locustName")`

Reply:
Oh, I didn't know that!! Thanks!